### DISCUSSION TITLE
Project 1 simpsons diversity

### DISCUSSION THREAD

**Discussion by Rawizard** [IBAHCM/RPiR#310] [Archive] [answered]
Hi @richardreeve & @soniamitchell ,

I am wondering for project 1 it says write a function that calculates Simpson diversity, now are you expecting us to use the vegan package in this or manually calculate it within the function using the diversity equation given in the lectures? 

Cheers

Jamie

**Comment by soniamitchell** [comment, marked as answer]
Hi @Rawizard, we'd like you to calculate diversity using the equation, or the description given in the Background section:
> Species richness is a very commonly used measure of diversity, which simply counts the number of non-zero species abundances; Simpson diversity is also commonly used: it simply calculates the species proportions from the counts (so the total sums to 1), squares them all individually, and add them all together – it tells us (amongst other things) how likely two randomly chosen individuals are to be from the same species.

**Comment by Rawizard** [comment]
Ok Thanks!!
